go.mod

```diff
@@ -9,5 +9,7 @@ require (
 	github.com/shirou/gopsutil/v3 v3.20.11
 	github.com/stretchr/testify v1.6.1
 	go.uber.org/multierr v1.5.0
+	golang.org/x/sync v0.0.0-20190423024810-112230192c58
+	gopkg.in/errgo.v2 v2.1.0
```

Comment:
Maybe `github.com/pkg/errors` is enough?

Reply:
yes